### ISSUE TITLE
Sockets error

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#6138]
Hi, @namangupta01 I'm seeing this error -- 

![image](https://user-images.githubusercontent.com/24359/63113022-a86c9f00-bf5f-11e9-95ab-997c5058c9eb.png)

Any thoughts on this? Thanks!

**Comment by jywarren** [comment]
Just on this page, but probably on any page? https://publiclab.org/wiki/sandbox-projects

**Comment by namangupta01** [comment]
Hey @jywarren, good to see you back. I think this error is same as https://github.com/publiclab/plots2/pull/5744#issuecomment-504156582 Can you have a look at nginx configuration at the server and check if it is updated as specified in the link or not?
Thanks

**Comment by namangupta01** [comment]
@jywarren can you check for the nginx conf and confirm if it is the case of nginx conf or not?

**Comment by namangupta01** [comment]
I guess we have correct nginx conf on unstable and stable servers

**Comment by jywarren** [comment]
OK, copying back in from #6226:

> Hi @namangupta01 ! I'm trying to test this out but am seeing an error similar to https://github.com/publiclab/plots2/issues/5980 on production:

```
Error during WebSocket handshake: Unexpected response code: 404

  | ActionCable.Connection.Connection.open | @ | cable-d30277fdde31fe…c5729884ec69.js:241
-- | -- | -- | --
  | (anonymous) | @ | cable-d30277fdde31fe…c5729884ec69.js:205
  | ActionCable.Connection.Connection.reopen | @ | cable-d30277fdde31fe…c5729884ec69.js:275
  | (anonymous)

```

```js
            this.webSocket = new ActionCable.WebSocket(this.consumer.url, protocols);
```

**Comment by icarito** [comment]
This is the current configuration in production:
```
location /cable {
        proxy_pass          http://localhost:4002;
        proxy_http_version 1.1;
        proxy_set_header Upgrade websocket;
        proxy_set_header Connection Upgrade;
}

location @plots {
      proxy_set_header        Host $host;
      proxy_set_header        X-Real-IP $remote_addr;
      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
      proxy_set_header        X-Forwarded-Proto $scheme;
 
      proxy_pass          http://localhost:4002; 
 
      proxy_redirect off; 
      proxy_buffering off; 
}
```
How to test it? Thanks!

**Comment by jywarren** [comment]
@namangupta01 can you offer some input here, thanks!

**Comment by jywarren** [comment]
@namangupta01 is this correct? What do we need to change? Thank you!

**Comment by namangupta01** [comment]
Let me check!

**Comment by namangupta01** [comment]
This seems to be correct. Is it not working? If not have you also restarted the `nginx`?

**Comment by jywarren** [comment]
this was already in place on production, so it was restarted several times.
Is there another reason this wouldn't be working? Thanks!

**Comment by jywarren** [comment]
Hi @namangupta01 just wondering if there's anything else we need to change?

**Comment by namangupta01** [comment]
As far as I think this is enough to make it work. Isn't it working now after restarting `nginx`?

**Comment by namangupta01** [comment]
If it's not working then let me know I will do some further digging.

**Comment by jywarren** [comment]
No, it seems not to be working. Nginx has been restarted several times
since this config was added. Can you do some additional digging? Thank you!

**Comment by namangupta01** [comment]
Hi @jywarren, I was digging into this. But found nothing wrong. Also, this not working on production but working on staging and unstable may be due to some other issue on the machine configuration or inbound traffic? What do you think?

**Comment by jywarren** [comment]
Hmm. @icarito, can we compare the nginx configuration to that on stable to
try to see why this isn't working?
